src/test/resources/pages/studentHomeHTML.html

```diff
@@ -31,6 +31,29 @@ <h1>
   </div>
   <br>
   <br>
+  <div class="panel panel-primary">
```

Comment:
This is also an unrelated change. Try to enable [GodMode](https://github.com/TEAMMATES/teammates/blob/master/docs/godmode.md) and run `StudentHomePageUiTest.java` several times, this change will disappear :)